sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Please specify the release date you expected by replacing the `Unreleased` , format is `yyyy-mm-dd`.